src/client/driver/get-offset-options.js

```diff
@@ -0,0 +1,20 @@
+import { positionUtils } from './deps/testcafe-core';
+
+
+function getDefaultAutomationOffsets (element) {
+    var rect = positionUtils.getElementRectangle(element);
+
+    return {
+        offsetX: Math.round(rect.width / 2),
+        offsetY: Math.round(rect.height / 2)
+    };
+}
+
+export default function getOffsetOptions (element, offsetX, offsetY) {

```

Comment:
we this in the [runner utils](https://github.com/DevExpress/testcafe/blob/master/src/client/runner/utils/mouse.js#L52)
It's better move it to the automation utils and use from legacy api and the new api, since automation scripts are for legacy and the new api